crates/cli/src/main.rs

```diff
@@ -1,56 +1,72 @@
 #![allow(unused)]
 
-use structopt::StructOpt;
+use thiserror::Error;
 
 use std::fs::File;
-use std::io::Read;
+use std::io::{self, Read};
 use std::path::PathBuf;
 use std::rc::Rc;
+use std::str::Utf8Error;
 
 use svm_gas::resolvers::ExampleResolver;
 use svm_gas::validate_wasm;
 use svm_gas::ProgramPricing;
 use svm_program::{Program, ProgramVisitor};
 
-#[derive(StructOpt, Debug)]
-#[structopt(name = "svm")]
-struct CLI {
-    #[structopt(long)]
-    wasm_file: Option<PathBuf>,
-}
-
-fn main() {
-    let opts = CLI::from_args();
+fn clap_app() -> clap::App<'static, 'static> {
+    use clap::*;
 
-    if let Some(wasm_file) = opts.wasm_file {
-        let file = File::open(&wasm_file);
-
-        if file.is_err() {
-            println!("File {:?} doesn't exist", &wasm_file);
-            return;
-        }
+    App::new("smwasm")
+        .version("1.0")
+        .author("The Spacemesh team")
```

Comment:
yes, we need to your this term in each crate (I've put my personal name because it was only me working basically on the project until now)

Reply:
Np